common4j/src/test/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthRequestHandlerTest.kt

```diff
@@ -104,23 +103,24 @@ class NativeAuthRequestHandlerTest {
             .clientId(clientId)
             .build()
 
-        nativeAuthRequestProvider.createSignUpStartRequest(
+        nativeAuthRequestProvider.createSignUpUsingPasswordStartRequest(
             commandParameters = commandParameters
         )
     }
 
-    @Test(expected = ClientException::class)
-    fun testSignUpStartWithEmptyPasswordShouldThrowException() {
-        val commandParameters = SignUpStartUsingPasswordCommandParameters.builder()
+    fun testSignUpStartWithEmptyPasswordShouldNotThrowException() {
+        val commandParameters = SignUpStartCommandParameters.builder()
             .platformComponents(mock<PlatformComponents>())
             .username(username)
             .password(emptyPassword)
             .clientId(clientId)
             .build()
 
-        nativeAuthRequestProvider.createSignUpUsingPasswordStartRequest(
+        val request = nativeAuthRequestProvider.createSignUpUsingPasswordStartRequest(
             commandParameters = commandParameters
         )
+
+        Assert.assertNotNull(request)
```

Comment:
Is this necessary (rather than just asserting no exception is thrown)? if so, let's add it everywhere in this class

Reply:
No, this is not necessary. This has been added as the annotation of this test is modified to remove expected exception.

Reply:
nit: you don't need this null check. Removing the expected = ClientException annotation should be enough. If an exception is thrown, the test will fail.

Reply:
I will explain my reasoning here. This assert is similar to "this page intentionally left blank" in print books. If this assert was not present then someone in future (me or other developers in the team) when reviewing this test might assume the test is incomplete. It could take them some time to understand this test does not require any modification. This assert statement shortens that time period.